single-line-diagram-core/src/main/resources/ConvergenceLibrary/components.json

```diff
@@ -410,5 +410,17 @@
       "styleClass" : "sld-arrow-in"
     } ],
     "styleClass" : "sld-arrow-q"
+  }, {
+    "type" : "LACK_VOLTAGE",
```

Comment:
I'm wondering if we should provide this component except in tests, as there's no way to add it to a network with the exisiting GraphBuilders. But that's maybe not the only component in that case in fact!

Reply:
How append lack-voltage component to existing ConvergenceLibrary only for tests ?